qualtran/bloqs/phase_estimation/phase_estimation_of_quantum_walk.ipynb

```diff
@@ -178,19 +204,7 @@
     "m_bits = int(np.ceil(np.log2(qlambda * np.pi * np.sqrt(2) / delta_E)))\n",
     "walk = get_walk_operator_for_hubbard_model(x_dim, y_dim, t, mu)\n",
     "circuit = cirq.Circuit(phase_estimation(walk, m=m_bits))\n",
-    "%time result = t_complexity_compat(circuit[1:-1])\n",
-    "print(result)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from qualtran.bloqs.phase_estimation.qubitization_qpe import _qubitization_qpe_hubbard_model_large\n",
-    "qpe = _qubitization_qpe_hubbard_model_large.make()\n",
-    "%time result = qpe.t_complexity()\n",
```

Comment:
Why are we removing this? AFAIR, `_qubitization_qpe_hubbard_model_large` actually returns a bloq which is equivalent to the cirq circuit being constructed above and the two t_complexity's should match.

Reply:
oh, I didn't realize that. I thought this was just the walk operator or sth. I think this was originally included to make sure the two disparate t_complexity code paths were executed; but now there's only one. We should remove the cirq stuff if we already have this as bloqs and bloq examples. I'll revert and open an issue